scripts/zones/RuAun_Gardens/mobs/Groundskeeper.lua

```diff
@@ -7,6 +7,26 @@ local ID = zones[xi.zone.RUAUN_GARDENS]
 -----------------------------------
 local entity = {}
 
+local despotPHTable =
+{
+    [ID.mob.DESPOT - 16] = ID.mob.DESPOT,
```

Comment:
Would it make more sense to use GetTableByIDs for this instead of +16 and so on?